gui/server.js

```diff
@@ -13,6 +16,18 @@ var app = express();
 var basedir = process.env.BASEDIR;
 var port = 3000;
 
+var logger = new (winston.Logger)({
+  transports: [
+    new (winston.transports.Console)({ level: 'warn' }),
+    new (winston.transports.File)({
+      filename: 'webui.log',
+      level: 'info'
+    })
+  ],
+  exitOnError: false,
+  emitErrs: false
```

Comment:
Full disclosure: I'm not exactly sure what the *emitErrs* and *exitOnError* options do...
Both of them sounded bad for a webserver, based on the description in the docs, so I disabled them for now.